cli-definition.json

```diff
@@ -27,6 +27,11 @@
       "use": "run",
       "example": "\n\tpgstream run --source postgres --source-url <source-postgres-url> --target postgres --target-url <target-postgres-url>\n\tpgstream run --source postgres --source-url <source-postgres-url> --target postgres --target-url <target-postgres-url> --snapshot-tables <schema.table> --reset\n\tpgstream run --source kafka --source-url <kafka-url> --target elasticsearch --target-url <elasticsearch-url>\n\tpgstream run --source postgres --source-url <postgres-url> --target kafka --target-url <kafka-url>\n\tpgstream run --config config.yaml --log-level info\n\tpgstream run --config config.env",
       "flags": [
+        {
+          "name": "profile",
+          "description": "Wether to expose a /debug/pprof/ endpoint on localhost:6060",
```

Comment:
why is there a "/" at the end?

Reply:
It's the root for other sub-endpoints (for example, `debug/pprof/block`). I'll remove it though.